### ISSUE TITLE
Transiflex not working with french

### ISSUE THREAD

**Issue by Maxtille** [niccokunzmann/mundraub-android#125]
I'm trying to contribute on the french translation of the app but transiflex doesn't propose french (even if it's already at a few percent translated) when I click on join the team. I think there  is a little problem ...

**Comment by niccokunzmann** [comment]
I added you to the project.
What I saw is that you requested ** French (France) (fr_FR) **
but only ** French (fr) ** is present at the moment.

You can translate it and report back if it works. I also did not get messages, so I did not know that something is going on there. Thanks for reporting here!

**Comment by Maxtille** [comment]
Hey ! Ok everything is fine, I just started working on the french version. It's my first-time with this tool and it's realy well made ! 
But there is some strings which are empty of words and I'm unable to provide a translation (because it says it's empty and I cannot save it) so that's why this is not 100 percent completed on some resources as f-droid full description.

**Comment by niccokunzmann** [comment]
Can you show me a link?

**Comment by Maxtille** [comment]
Yep, per exemple the sixth box of this page contains nothing : https://www.transifex.com/mundraub-android/mundraub-android-app/translate/#fr/fdroid-full-description-txt/152622627

**Comment by niccokunzmann** [comment]
That is true. It is the `<ul>` tag. I translated it by clicking on it. Then, it was copied to the bottom. It does not need translation though.

**Comment by Maxtille** [comment]
Ok I got it, but is it preferable to copy all of the tags ? As I just translated the words and didn't bring all those tags in my translation,but I could if it's usefull.

**Comment by niccokunzmann** [comment]
Yes, that is useful for the styling. But I can not test if HTML tags for styling work. I will see when f-droid deploys it. In the mean time: if you are fine with it, I am fine with it.

**Comment by niccokunzmann** [comment]
Thanks for translating so much. If you like to see the result: I uploaded new commits and the latest release should include your translations.

**Comment by niccokunzmann** [comment]
@Maxtille Would you say there is something we need to do with this issue?

**Comment by Maxtille** [comment]
Issue #125 can be definitively closed for me as we are trying to find a solution on #130 ;)
